datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/transform.py

```diff
@@ -88,6 +93,10 @@ def compile_transformer(self, config):
         if not isinstance(metric_type, str):
             raise TypeError('field `type` must be a string')
 
+        custom_transformer = config.pop('__transformer__', None)
+        if custom_transformer:
+            return metric_type, custom_transformer
```

Comment:
Let's make this first:

```suggestion
        custom_transformer = config.pop('__transformer__', None)
        if custom_transformer:
            return None, custom_transformer
```